bookdown/02-model-optimization/00-model-optimization.Rmd

```diff
@@ -1 +1,25 @@
 # Model Optimization {#model-optim}
+This chapter will introduce different approaches to model opitmization. 
+
+**Model Tuning**
+The first sub-section discusses "model tuning".  
+Many machine learning algorithms have hyperparameters that need to be set. 
+If selected by the user they can be specified as explained on the tutorial page on learners – simply pass them to makeLearner(). 
+Often suitable parameter values are not obvious and it is preferable to tune the hyperparameters
+Tuning hyperparameters implies automatic identification of values that lead to the best performance.
+In order to tune a machine learning algorithm, you have to specify: (1) the search space;(2) the optimization algorithm (aka tuning method) and (3) an evaluation method, i.e., a resampling strategy and a performance measure.
+In summary, the sub-chapter on tuning illustates hyperparamater selection, how to pick an optimizing algorithm and how to automate tuning.  
+
+**Feature Selection** 
+The second part of this chapter explains "feature selection". 
+The objective of feature selection is to fit the sparce dependent of a model on a subset of available data features in the most suitable manner.
+Feature selection can enhance the interpretability of the model, speed up the learning process and improve the learner performance.
+Different approaches exist to identify the relevant features.
+The chapter on feature selection introduces two distinct approaches to filtering, namely oon the one hand "filtering" and on the other hand  an approach referred to as "feature subset selection" or Wrapper methods".
+
+**Nested Resampling**
+The third section is about resampling strategies, specifically nested resampling. 
+Resampling strategies are usually used to assess the performance of a learning algorithm:The entire data set is (repeatedly) split into training sets D∗b and test sets D∖D∗b, b=1,…,B.
+The learner is trained on each training set, predictions are made on the corresponding test set (sometimes on the training set as well) and the performance measure S(D∗b,D∖D∗b) is calculated.
+Then the B individual performance values are aggregated, most often by calculating the mean. Various different resampling strategies exist, for example cross-validation and bootstrap, to mention just two popular approaches.
```

Comment:
```suggestion
Then the B individual performance values are aggregated.
Various different resampling strategies exist, for example cross-validation and bootstrap, to mention just two popular approaches.
```

One sentence per line

We should leave the selection of the aggregation method to the user.